developer_setup.md

```diff
@@ -299,3 +299,32 @@ If this happens, you may be missing developer tools in your OS X. Try to install
 
 If this happens it may be a Bundler issue. Try running `bundle config specific_platform true`
 and re-try.
+
+* Can't install any gems during bundle install
```

Comment:
Sorry, I don't .. but I should be able to re-try in september, so I'll do a follow-up PR then (I hope :)).